src/main/java/mate/academy/service/UserService.java

```diff
@@ -8,13 +8,22 @@ public class UserService {
             new User("alice@i.ua", "1234")
     };
 
+    public static User[] getUsers() {
```

Comment:
you don't need to create extra method and make it public, you can just refer to the users array as it's static